docs/entities/servicebinding.md

```diff
@@ -0,0 +1,38 @@
+# ServiceBinding
+
+A Service Binding represents the secrets along with the Applications which are to bound with it. Service Bindings may explicitly request an Application by Name or by LabelSelector. LabelSelector can match more than one resource. Service Binding resource is being created by the `ServiceClaim` controller. Service Binding is being performed by `Primaza Application Agent` , and the agent should be deployed in the Application namespace where the workloads are to be bound. When a Service Binding is created (or updated) into an Application namespace, the Application agent has to get the data from the `SEDs` and project them into applications matching the Application specification from Service Binding instance. Currently the secret data is being projected as volume mounts. `SERVICE_BINDING_ROOT` points to the environment variable in the container which is used as the volume mount path.  In the absence of this environment variable, `/bindings` is used as the volume mount path. Please Refer to https://github.com/servicebinding/spec#reconciler-implementation for more information.
+
+
+## Specification
+
+The definition of a Service Binding can be obtained directly from our [ServiceBinding CRD](../../config/crd/bases/primaza.io_servicebindings.yaml). The specification contains two required properties:
+
+`ServiceEndpointDefinitionSecret`: ServiceEndpointDefinitionSecret is the name of the secret to project into the application. This property is required.
+`Application`: 	Application resource to inject the binding info. It could be any process running within a container. A `ServiceBinding` **MAY** define the application reference by-name or by-[label selector][ls]. A name and selector **MUST NOT** be defined in the same reference.
```

Comment:
you can just say they are mutually exclusive

Reply:
Makes sense